CHANGELOG.md

```diff
@@ -1,6 +1,23 @@
 Changelog
 =========
 
+## 6.1.8 (2020-10-07)
+
+### Enhancements
```

Comment:
Are these more bug fixes than enhancements? If we classify them as enhancements, we should make this 6.2.0.